Installation/include/CGAL/config.h

```diff
@@ -590,6 +590,16 @@ using std::max;
 #define CGAL_CAN_USE_CXX11_ATOMIC
 #endif
 
+#ifndef CGAL_HAS_THREADS
+inline bool is_currently_single_threaded(){ return true; }
+#elif __has_include(<sys/single_threaded.h>)
+#include <sys/single_threaded.h>
+inline bool is_currently_single_threaded(){ return ::__libc_single_threaded; }
+#else
+/* This is the conservative default */
+inline bool is_currently_single_threaded(){ return false; }
+#endif
```

Comment:
```suggestion
namespace CGAL {
#ifndef CGAL_HAS_THREADS
  inline bool is_currently_single_threaded(){ return true; }
#elif __has_include(<sys/single_threaded.h>)
#  include <sys/single_threaded.h>
  inline bool is_currently_single_threaded(){ return ::__libc_single_threaded; }
#else
/* This is the conservative default */
  inline bool is_currently_single_threaded(){ return false; }
#endif
}
```

Reply:
Ah, right, the namespace. I'll avoid including a system header from inside namespace CGAL though :wink: